### PR TITLE
Athlete functionality

### PR DESCRIPTION
**Pull request by DHowell1150**
Changed `Height` attribute to 'feet' and 'inches' which reached to the following folders:
- Model validations
- schema
- athlete_create_spec
- athletes/create.html.erb 

I seeded a user and two athletes associated with that User so we could use Rails s to test user functionality. 

Had a user log in in athlete_create_spec before testing functionality for creating athlete.  This pulled it all together.  
Deleted @athlete in before block.

### CODE REVIEW

File: app/models/athlete.rb
Comment (s2an) [inline app/models/athlete.rb:1]: I am glad this was easy to implement. It sure beats having the user have to do math!

File: app/views/athletes/new.html.erb
Comment (s2an) [inline app/views/athletes/new.html.erb:1]: I think this works for now (styling with the all caps). Later on making it pretty will be fun, but there are more important fish to fry.

File: app/views/athletes/show.html.erb
Comment (s2an) [inline app/views/athletes/show.html.erb:1]: I found it is helpful to put a generic title at first:

<h1>Athlete Show Page</h1>

That way when you are moving forward and hoping around in the repo in VS code it is really easy to see where you are at.

**Comment by s2an** [comment]
This all looks great!

File: app/views/athletes/show.html.erb
Comment (DHowell1150) [reply app/views/athletes/show.html.erb:1]: But a user wouldn't know what a show page is, that's why I put the name up there.  I figured maybe we switch it to "My Dashboard" or something that a non-coder user/athlete would recognize?

**Pull request merged**